groovy-android-gradle-plugin/src/test/groovy/groovyx/functional/FullCompilationSpec.groovy

```diff
@@ -120,11 +121,11 @@ class FullCompilationSpec extends AndroidFunctionalSpec implements AndroidFileHe
     then:
     noExceptionThrown()
     file('build/outputs/apk/debug/test-app-debug.apk').exists()
-    file('build/intermediates/javac/debug/compileDebugJavaWithJavac/classes/groovyx/test/MainActivity.class').exists()
-    file('build/intermediates/javac/debugAndroidTest/compileDebugAndroidTestJavaWithJavac/classes/groovyx/test/AndroidTest.class').exists()
+    file("build/intermediates/javac/debug/compileDebugJavaWithJavac/classes/groovyx/test/MainActivity.class").exists()
+    file("build/intermediates//javac/debugAndroidTest/compileDebugAndroidTestJavaWithJavac/classes/groovyx/test/AndroidTest.class").exists()
```

Comment:
Was this extra slash intentional?

Reply:
No, but doesn't effect anything. I'll clean it up at a later time. Once merged snapshot should be releasing again.